types/vote.go

```diff
@@ -74,7 +76,7 @@ func (vote *Vote) CommitSig() *CommitSig {
 
 func (vote *Vote) SignBytes(chainID string) []byte {
 	// [peppermint] converted from amino to rlp
-	bz, err := rlp.EncodeToBytes(CanonicalizeVote(chainID, vote))
+	bz, err := cdc.MarshalBinaryLengthPrefixed(CanonicalizeVote(chainID, vote))
```

Comment:
Why did we remove RLP from here?

Reply:
@vaibhavchellani We are not using RLP encoding anymore for checkpoint since we don't need to verify the vote on the contract. We can use `side-tx` data to sign on checkpoint data.

Reply:
Gotcha